apps/andi/assets/package.json

```diff
@@ -6,7 +6,6 @@
   },
   "dependencies": {
     "cacache": "^15.2.0",
-    "material-icons": "^1.11.8",
```

Comment:
There's kind of a lot of mentions of material icons in the code still, sure this removal is okay? Looks like it's just what we call the classes for css (and one spot that mentions it for a font `font-family: 'Material Icons';`), but figure'd I'd ask

Reply:
Should no longer be necessary because we're self-hosting the icons. The material-icons package was supposed to be a way to self host the icons without doing all the work ourselves, but I couldn't get it working. If you want to double check that it works, you can checkout my branch, go into apps/andi/assets and run and npm clean-install. Then start up andi as normal and it should still show all the material icons, such as on the transformation form header, etc.